### DISCUSSION TITLE
No longer reporting consistently

### DISCUSSION THREAD

**Discussion by kmatwill** [alandtse/tesla#591] [General]
Hi, I have done a few updates, but at some point my tesla information stopped reporting until I actually take some action with the vehicle. It's like the polling feature isn't working. Anyone else seen this? fix? If not, I'll just blow it away and reinstall.

**Comment by alandtse** [comment]
Is your car asleep? https://github.com/alandtse/tesla/wiki/Polling-policy

**Reply by kmatwill** [reply]
Got it figured out. Had to delete the integration and reinstall with a new sso refresh token.
